nomad/utils/text.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import re
+import unicodedata
+
+
+_punct_re = re.compile(r'[\t !"#$%&\'()*\-/<=>?@\[\\\]^_`{|},.]+')
+
+
+def slugify(text):
+    """ Returns a ASCII-only slug version of the considered text. """
+    result = []
+    for word in _punct_re.split(text.lower()):
+        word = unicodedata.normalize('NFKD', word).encode('ascii', 'ignore').decode('ascii')
```

Comment:
Fair enough. :wink: I'll remove the `slugify` helper, but the proper validations ruls will be added in a subsequent PR.